class/defaults.yml

```diff
@@ -18,6 +18,7 @@ parameters:
     operator:
       manifest_version: v${lieutenant:images:operator:version}
       manifest_url: https://raw.githubusercontent.com/projectsyn/lieutenant-operator/${lieutenant:operator:manifest_version}/config
+      lieutenant_api_url: https://${lieutenant:api:ingress:host}
```

Comment:
Is this ever different than `${lieutenant:api:ingress:host}`?

Reply:
Probably not.